plugins/woocommerce/includes/class-wc-ajax.php

```diff
@@ -3099,6 +3100,7 @@ public static function toggle_gateway_enabled() {
 					$gateway->update_option( 'enabled', 'no' );
 				}
 
+				do_action( 'woocommerce_update_options' );
```

Comment:
If we go with my other recommendation, we can probably remove the 2 additions in this file.  If not, we may want to pass additional data to this action to make it more useful to consumers.

Reply:
Scratch my previous comment and leave this one.